### PR TITLE
Flex Pricing files cleanup

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Flex Pricing files cleanup plus one unrelated edit to sumo-logic-ui

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/manage/partitions/flex/faq.md [hunk 1]
```diff
@@ -81,9 +83,11 @@ We recommend you configure partitions to have less than 5 TB per day flowing int
...
+You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating or updating the partition. If, out of 10 partitions, you exclude two partitions, any query that does not reference `_index` or `_view` will only consider the included partitions. This reduces the amount of data scanned and lowers the cost.
+
+When partitions are marked as included and `_index` or `_view` is not referenced in the query, all included partitions will be considered by default. Default scope is also useful when `AND` or `OR` conditions are used in the query with `_index`. For example, consider you have three partitions: Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:
 
-- When you run the query without referring to `_index`, for example `error | count`, only Partition B and Partition C will be considered for the query, as Partition A is excluded from the default scope. 
+- When you run the query without referring to `_index`, for example `error | count`, only Partition B and Partition C will be considered for the query, as Partition A is excluded from the default scope.
```
Comment (kimsauce) [inline docs/manage/partitions/flex/faq.md:90]:
```suggestion
- When you run the query without referring to `_index`, for example `error | count`, only Partition B and Partition C will be considered for the query, as Partition A is excluded from the default scope. 
```

File: docs/manage/partitions/flex/faq.md [hunk 2]
```diff
@@ -77,11 +79,13 @@ We recommend you configure partitions to have less than 5 TB per day flowing int
...
 
-When partitions are marked as included and `_index`/`_view` is not referenced in the query, all the included partitions will be considered for the search query by default. Default scope can also come into use when `AND` or `OR` conditions are referred to in the query with `_index`. For example, consider you have three partitions namely, Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:
+You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating or updating the partition. If, out of 10 partitions, you exclude two partitions, any query that does not reference `_index` or `_view` will only consider the included partitions. This reduces the amount of data scanned and lowers the cost.
```
Comment (jpipkin1) [inline docs/manage/partitions/flex/faq.md:86]: @kimsauce, this paragraph repeats the last half of the previous paragraph.

Comment (kimsauce) [reply docs/manage/partitions/flex/faq.md:86]: YIKES! Good catch. This was a merge conflict error on my part. I'll fix.

**Review by jpipkin1 — APPROVED** [review]
Very nice, @kimsauce! I have only one comment.

File: docs/manage/partitions/flex/faq.md [hunk 3]
```diff
@@ -77,11 +79,13 @@ We recommend you configure partitions to have less than 5 TB per day flowing int
 * Reduces the cost of each query.
 * Accelerates queries, as less data has to be scanned.
 
-## How do I optimize my query using default scope?
+## How can I optimize my query using default scope?
 
 Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, if a query needs to run through 10 partitions, which consumes about 10 GB of search data, narrowing the search query using the default scope can improve the query performance and reduce scan cost. You can modify the default scope by selecting or deselecting the **Include this partition in default scope** checkbox when creating/updating your partition. Let's say that out of 10 partitions, you excluded two partitions. Now, when you run a query that does not have `_index` / `_view` term referenced in the query, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.
```
Comment (jpipkin1) [inline docs/manage/partitions/flex/faq.md:84]:
```suggestion
Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, if a query needs to run through 10 partitions, which consumes about 10 GB of search data, narrowing the search query using the default scope can improve the query performance and reduce scan cost. 
```

File: docs/manage/partitions/flex/faq.md (hunk 2 above)
Comment (jpipkin1) [reply docs/manage/partitions/flex/faq.md]: @kimsauce, I removed the repeating paragraph.

Comment (kimsauce) [reply docs/manage/partitions/flex/faq.md]: thank you!

**Pull request merged**